core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-card/dot-pages-card.component.scss

```diff
@@ -34,11 +34,19 @@
 
     .dot-pages-favorite-card-content__title {
         font-weight: bold;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
     }
 
     .dot-pages-favorite-card-content__subtitle {
         overflow: hidden;
         text-overflow: ellipsis;
         white-space: nowrap;
     }
+
+    .p-card .p-card-body {
+        box-shadow: 0px -15px 20px -20px rgba(0, 0, 0, 0.3);
```

Comment:
use the `$md-shadow-x` in `_variables.scss`

Reply:
this shadow is only used here and nowhere else, it's negative and the ones in variables are positive
![image](https://user-images.githubusercontent.com/37185433/219202175-1cd19129-3e87-4e71-883a-4a5e0c0152c8.png)